warehouse/query-core/src/test/java/datawave/query/util/VisibilityWiseGuysIngest.java

```diff
@@ -82,9 +82,17 @@ public static void writeItAll(Connector con, WhatKindaRange range) throws Except
             mutation.put(datatype + "\u0000" + corleoneUID, "BIRTHDAY.3" + "\u0000" + "4", columnVisibilityItalian, timeStamp, emptyValue);
             mutation.put(datatype + "\u0000" + corleoneUID, "BIRTHDAY.4" + "\u0000" + "5", columnVisibilityItalian, timeStamp, emptyValue);
             mutation.put(datatype + "\u0000" + corleoneUID, "BIRTHDAY.5" + "\u0000" + "22", columnVisibilityItalian, timeStamp, emptyValue);
+            // to test whether sparse fields are properly ignored while grouping, leave dependents field blank for group 0 and 3
+            // mutation.put(datatype + "\u0000" + corleoneUID, "DEPENDENTS.0" + "\u0000" + "2", columnVisibilityItalian, timeStamp, emptyValue);
```

Comment:
if you don't need this commented out code, please remove

Reply:
Done